rflx/pyrflx/typevalue.py

```diff
@@ -66,6 +70,28 @@ def construct(cls, vtype: Type) -> "TypeValue":
             return OpaqueValue(vtype)
         raise ValueError("cannot construct unknown type: " + type(vtype).__name__)
 
+    @staticmethod
+    def convert_bytes_to_bitstring(msg: bytes) -> str:
```

Comment:
You could use `format(int.from_bytes(msg, "big"), f"0{len(msg) * 8b")` for this.